packages/app/src/components/DBRowTable.tsx

```diff
@@ -23,6 +23,7 @@ import {
   extractColumnReference,
   JSDataType,
 } from '@/clickhouse';
+import { useExplainQuery } from '@/hooks/useExplainQuery';
```

Comment:
unused import?

Reply:
No. It links to the new useExplainQuery hook I introduced. https://github.com/hyperdxio/hyperdx/pull/538/files/ec937e1522271aadfa9e77fbb80084ef65942cf4#diff-cb81c6da9e9cb0767375c8b8d50af677205f01406ebb236320d47a37ab10b9a6

Reply:
🤦 don't hire me as a linter that's for sure